sources/service-kubernetes/main.tf

```
@@ -32,10 +32,20 @@ locals {
   docker_image_active  = var.docker_image_active == "" ? local.docker_image_active_calculated : var.docker_image_active
   docker_image_passive = var.docker_image_passive == "" ? local.docker_image_passive_calculated : var.docker_image_passive
 
-  ingresses_passive = [for x in var.ingresses : {
+  ingresses_passive_tmp = [for x in var.ingresses : {
```

I find `tmp` a little misleading here. Maybe we can consider something like `intermediate`, instead?
```suggestion
  ingresses_passive_intermediate = [for x in var.ingresses : {
```